gulpfile.js

```diff
@@ -3,40 +3,75 @@ var deployDir = 'docs';
 
 var gulp = require('gulp');
 var clean = require('gulp-clean');
+var concat = require('gulp-concat');
 var jshint = require('gulp-jshint');
+var uglify = require('gulp-uglify');
+var babel = require('gulp-babel');
+var inject = require('gulp-inject');
+var minifyHtml = require('gulp-minify-html');
+var minifyCss = require('gulp-clean-css');
+var browserSync = require('browser-sync');
+var injectVersion = require('gulp-inject-version');
 
-gulp.task('default', ['copy']);
+gulp.task('default', ['lint', 'html']);
```

Comment:
The html task no longer exists? So now the default gulp task just lint's your code.